packages/frontend/src/components/ExportResults/index.tsx

```diff
@@ -106,7 +115,20 @@ const ExportResults: FC<ExportResultsProps> = memo(
                         attachmentDownloadName: chartName
                             ? `${chartName}_${formatDate(new Date())}`
                             : undefined,
-                    });
+                    };
+
+                    // Use embed-specific endpoint if in embed context
+                    return isEmbedContext
+                        ? scheduleEmbedDownloadQuery(
+                              projectUuid,
+                              queryUuid,
+                              downloadOptions,
+                          )
+                        : scheduleDownloadQuery(
```

Comment:
What's the difference between the embed and standard query? Could we use the same endpoint for both?